idaes/models/unit_models/tests/test_feed_flash.py

```diff
@@ -273,7 +271,7 @@ def test_initialize(self, iapws):
     @pytest.mark.skipif(solver is None, reason="Solver not available")
     @pytest.mark.component
     def test_solve(self, iapws):
-        results = solver.solve(iapws)
+        results = solver.solve(iapws, tee=True)
```

Comment:
Should we be setting `tee=True` in a test? I suppose it can help if the test fails, but otherwise it will just get hidden by pytest.